### PR TITLE
antisnake

### PR DESCRIPTION
**Pull request by kkiirill**
[DEMO LINK](https://kkiirill.github.io/layout_antisnake/)
[TEST REPORT LINK](https://kkiirill.github.io/layout_antisnake/report/html_report/)

### CODE REVIEW

File: src/styles/main.scss [hunk 1]
```diff
@@ -1,3 +1,53 @@
...
+  }
+  &__card--4 {
+    background: rgb(102,0,0);
+  }
+  &__card--5 {
+    background: rgb(51,0,0);
+  }
+  &__card--6 {
+    background: rgb(0,0,0);
+  }
```
Comment (Olexandr6) [inline src/styles/main.scss:52]: Let's use `@each in` loop here, and try to implement this logic without any extra classes)

*Hint*: use `nth-child` pseudoclass.

File: src/styles/main.scss [hunk 2]
```diff
@@ -1,3 +1,53 @@
...
+    grid-template-rows: repeat(2, 1fr);
+  }
+
+  @media (max-width: 450px) {
+    grid-template-columns: 1fr;
+    grid-template-rows: 1fr;
+  }
+
+  &__card {
+    min-height: 300px;
```
Comment (Olexandr6) [inline src/styles/main.scss:25]: Create a variable for this value, you use it in the several places

File: src/styles/main.scss [hunk 3]
```diff
@@ -1,3 +1,53 @@
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
 }
+
+.box {
```
Comment (Olexandr6) [inline src/styles/main.scss:9]: Create a separate styles file for each BEM block

File: src/styles/main.scss [hunk 4]
```diff
@@ -1,3 +1,53 @@
...
+  }
+
+  @media (max-width: 450px) {
+    grid-template-columns: 1fr;
+    grid-template-rows: 1fr;
+  }
+
+  &__card {
+    min-height: 300px;
+    font-family: Arial, Helvetica, sans-serif;
```
Comment (Olexandr6) [inline src/styles/main.scss:26]: Set `font-family` in the root selector

**Review by Olexandr6 — CHANGES_REQUESTED** [review]
There must be 2 blocks in the row, start from 600px screen width:
image

There must be 3 blocks in the row, start from 900px screen width:
image

Check examples [here](https://github.com/mate-academy/layout_antisnake/tree/master/reference)

File: src/styles/box.scss [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+  display: grid;
+  grid-template-columns: repeat(3, minmax($size, 1fr));
+  height: 100vh;
+
+  @media (max-width: 899px) {
+    grid-template-columns: repeat(2, 1fr);
+    grid-template-rows: repeat(2, 1fr);
+  }
+
+  @media (max-width: 600px) {
```
Comment (illiarerun) [inline src/styles/box.scss:11]: that's what I see on 600px breakpoint, it should be 2 blocks, hint: you should lower max-width value a bit
image

File: src/styles/box.scss [hunk 6]
```diff
@@ -0,0 +1,31 @@
+.box {
+  display: grid;
+  grid-template-columns: repeat(3, minmax($size, 1fr));
+  height: 100vh;
+
+  @media (max-width: 899px) {
+    grid-template-columns: repeat(2, 1fr);
+    grid-template-rows: repeat(2, 1fr);
```
Comment (illiarerun) [inline src/styles/box.scss:8]: redundant
```suggestion
```

File: src/styles/box.scss [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+  }
+
+  @media (max-width: 600px) {
+    grid-template-columns: 1fr;
+    grid-template-rows: 1fr;
```
Comment (illiarerun) [inline src/styles/box.scss:13]: redunant
```suggestion
```

File: src/styles/box.scss [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+  &__card {
+    min-height: $size;
+    font-size: 100px;
+    color: white;
+    display: flex;
+    text-align: center;
+    align-items: center;
+    justify-content: center;
+
+    @each $a, $b in $colors {
```
Comment (illiarerun) [inline src/styles/box.scss:25]: what are `a` and `b`, be more specific with namings

**Review by illiarerun — CHANGES_REQUESTED** [review]
few more fixes left:

**Review by illiarerun — APPROVED** [review]

**Pull request closed without merging**